Snowflake.Data.Tests/Snowflake.Data.Tests.csproj

```diff
@@ -1,118 +1,36 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <ProjectGuid>{97E793E8-62A4-4FF2-9406-F5277269EEC2}</ProjectGuid>
-    <OutputType>Library</OutputType>
-    <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>Snowflake.Data.Tests</RootNamespace>
-    <AssemblyName>Snowflake.Data.Tests</AssemblyName>
-    <TargetFrameworkVersion>v4.6</TargetFrameworkVersion>
-    <FileAlignment>512</FileAlignment>
-    <TargetFrameworkProfile />
+    <TargetFrameworks>netcoreapp2.0;net46</TargetFrameworks>

```

Comment:
Same question on the target framework: do we need to explicitly target 4.6**.1**, or will targeting 4.6 ensure all customers have 4.6.1 at least?

Reply:
Same question from me. If our library against 4.6, so that everyone using 4.6 or 4.6.1 should be able to use our connector? Do I miss anything?